examples/airflow/README.md

```diff
@@ -1,43 +1,122 @@
+    catchup=False,
+    is_paused_upon_creation=True,
+    default_args=default_args,
+    description='Loads new deliveries for the week.'
+)
+
+t1 = PostgresOperator(
+    task_id='if_not_exists',
+    postgres_conn_id='food_delivery_db',
+    sql='''
+    CREATE TABLE IF NOT EXISTS delivery_7_days (
+      order_id            INTEGER REFERENCES orders(id),
+      order_placed_on     TIMESTAMP NOT NULL,
+      order_dispatched_on TIMESTAMP NOT NULL,
+      order_delivered_on  TIMESTAMP NOT NULL,
+      customer_email      VARCHAR(64) NOT NULL,
+      customer_address    VARCHAR(64) NOT NULL,
+      discount_id         INTEGER REFERENCES discounts(id),
+      menu_id             INTEGER REFERENCES menus(id),
+      restaurant_id       INTEGER REFERENCES restaurants(id),
+      restaurant_address  VARCHAR(64) NOT NULL,
+      menu_item_id        INTEGER REFERENCES menu_items(id),
+      category_id         INTEGER REFERENCES categories(id),
+      driver_id           INTEGER REFERENCES drivers(id)
+    );''',
+    dag=dag
+)
+
+t2 = PostgresOperator(
+    task_id='tuncate',
+    postgres_conn_id='food_delivery_db',
+    sql='TRUNCATE TABLE delivery_7_days;',
+    dag=dag
+)
+
+t3 = PostgresOperator(
+    task_id='insert',
+    postgres_conn_id='food_delivery_db',
+    sql='''
+    INSERT INTO delivery_7_days (order_id, order_placed_on, order_dispatched_on, order_delivered_on, customer_email, customer_address, discount_id, menu_id, restaurant_id, restaurant_address, menu_item_id, category_id, driver_id)
+      SELECT o.order_id, o.placed_on AS order_placed_on,
+        (SELECT transitioned_at FROM order_status WHERE order_id = o.order_id AND status = 'DISPATCHED') AS order_dispatched_on,
+        (SELECT transitioned_at FROM order_status WHERE order_id = o.order_id AND status = 'DELIVERED') AS order_delivered_on,
+        c.email AS customer_email, c.address AS customer_address, o.discount_id, o.menu_id, o.restaurant_id, r.address, o.menu_item_id, o.category_id, d.id AS driver_id
+        FROM orders_7_days AS o
+       INNER JOIN order_status AS os
+          ON os.order_id = o.order_id
+       INNER JOIN customers AS c
+          ON c.id = os.customer_id
+       INNER JOIN restaurants AS r
+          ON r.id = os.restaurant_id
+       INNER JOIN drivers AS d
+          ON d.id = os.driver_id
+       WHERE os.transitioned_at >= NOW() - interval '7 days'
+    ''',
+    dag=dag
+)
+
+t1 >> t2 >> t3
+```
 
-To start, you'll see a list of DAGs. The Airflow [scheduler](https://airflow.apache.org/docs/stable/scheduler.html) will search for DAGs under `dags/`, but also periodically scan the directory for new workflows. To the left of each DAG, the toggle has been switch to `ON`. This means that the DAG is ready to run. By default, DAGs are `OFF` ([an option](https://github.com/apache/airflow/blob/master/airflow/config_templates/default_airflow.cfg#L111) that users can easily configure).
+## Step 3: Explore inter-DAG Dependencies
```

Comment:
```suggestion
## Step 3: View collected metadata
To see the metadata that Marquez collected from airflow, browse back to the Marquez UI by visiting http://localhost:3000 . Use the search bar in the upper right corner and search for the job `etl_delivery_7_days`. Click on the item that appears in the dropdown.

Note: If no item is in the dropdown, check to see if airflow has successfully executed the DAG.

**Congrats**! You are now collecting metadata with marquez! Explore the other DAGs in the the examples/airflow directory to build expressive data lineage views.
```

Reply:
I'm not sure where to lead them so they can discover how common industry problems are solvable with marquez.